Assets/Scripts/Controllers/TileSpriteController.cs

```diff
@@ -75,8 +80,20 @@ protected override void OnChanged(Tile tile)
             Debug.ULogErrorChannel("TileSpriteController", "tileGameObjectMap's returned GameObject is null -- did you forget to add the tile to the dictionary? Or maybe forget to unregister a callback?");
             return;
         }
-        
-        tile_go.GetComponent<SpriteRenderer>().sprite = SpriteManager.current.GetSprite("Tile", tile.Type.Name);
+
+        //TODO Evaluate this criteria and naming schema!
```

Comment:
The truth is, I'm not really sure how this should look until it's been in the hands of a few users. Is 100 times entering a tile so rare it will never happen, or happen all the time? I really don't know. We could discuss this further if desired, I'm just not sure if that would help significantly.